crates/hyperdrive-math/src/lp.rs

```diff
@@ -353,6 +412,164 @@ impl State {
             ))
             .unwrap()
     }
+
+    /// Calculates the number of share reserves that are not reserved by open
+    /// positions.
+    fn calculate_idle_share_reserves(&self) -> FixedPoint {
+        let long_exposure = self.long_exposure().div_up(self.vault_share_price());
+        let idle_shares = {
+            if self.share_reserves() > long_exposure + self.minimum_share_reserves() {
+                return self.share_reserves() - long_exposure - self.minimum_share_reserves();
+            }
+            fixed!(0)
```

Comment:
I think it's more clear to return in each condition and not assign to a variable. e.g.:
```rs
if self.share_reserves() > long_exposure + self.minimum_share_reserves() {
  self.share_reserves() - long_exposure - self.minimum_share_reserves()
} else {
  fixed!(0)
}
```

Reply:
yup